src/mochiweb_socket_server.erl

```diff
@@ -60,12 +61,15 @@ set(Name, Property, _Value) ->
                           [Name, Property]).
 
 stop(Name) when is_atom(Name) orelse is_pid(Name) ->
-    gen_server:call(Name, stop);
+  gen_server:call(Name, stop);
 stop({Scope, Name}) when Scope =:= local orelse Scope =:= global ->
     stop(Name);
 stop(Options) ->
     State = parse_options(Options),
     stop(State#mochiweb_socket_server.name).
+stop(Name, Timeout) when is_atom(Name) orelse is_pid(Name) andalso is_integer(Timeout) ->
```

Comment:
andalso has precedence over orelse. Therefore, this guard will be true for:
`is_atom(Name)` (independently of `Timeout`'s type)
or
`is_pid(Name) andalso is_integer(Timeout)`

Is that the expected behaviour? I believe it should be:
```suggestion
stop(Name, Timeout) when (is_atom(Name) orelse is_pid(Name)) andalso is_integer(Timeout) ->
```